### ISSUE TITLE
Reinforcement Learning goals #1

### ISSUE THREAD

**Issue by Riotpiaole** [jQwotos/HootMentor#1]
Fix the model into more suitable chatbot for recommendation

**Comment by jQwotos** [comment]
We also need better training data, that results into the bot giving relevant responses. Then we can populate the responses with proper data (such as income, etc.). Since there isn't much point in training the bot each time we want it to give a new response to specific jobs. Similar to how google does their card system.

**Comment by jQwotos** [comment]
A quick note on that, I would recommend that we just take a quick look at some training data, finding a couple keywords manually then using a thesaurus.com scraper to build more keywords, replace all those keywords in the training data with a tag.

**Comment by Riotpiaole** [comment]
Yeah, is a great idea you can create anther issue and create a new branch.
